mozilla-release/browser/components/preferences/in-content/privacy.js

```diff
@@ -348,6 +348,7 @@ var gPrivacyPane = {
     document.getElementById("forgetMode").hidden = !autoForgetTabs.hasDatabase;
 #endif
 
+    this.blockAutoplayReadPrefs();
```

Comment:
@tsl143 die Frage ist: are you sure you do not need to enable those lines of code as well?

 Preferences.get("media.autoplay.default").on("change",
     gPrivacyPane.blockAutoplayReadPrefs.bind(gPrivacyPane));